src/core/core/network.cpp

```diff
@@ -340,6 +340,9 @@ namespace dsn
     {
         if (is_client())
         {
+            /* TODO(qinzuoyan): now that we will always call on_recv_reply() to notice the failure,
```

Comment:
What do you mean by "always call on-recv-reply() to notify the failure"?  why no need to reconnect?